doc/classes/Dictionary.xml

```diff
@@ -148,6 +148,19 @@
 				Constructs an empty [Dictionary].
 			</description>
 		</constructor>
+		<constructor name="Dictionary">
+			<return type="Dictionary" />
+			<param index="0" name="base" type="Dictionary" />
+			<param index="1" name="key_type" type="int" />
+			<param index="2" name="key_class_name" type="StringName" />
+			<param index="3" name="key_script" type="Variant" />
+			<param index="4" name="value_type" type="int" />
+			<param index="5" name="value_class_name" type="StringName" />
+			<param index="6" name="value_script" type="Variant" />
+			<description>
+				Creates a typed dictionary from the [param base] dictionary.
```

Comment:
This is _way too succinct_, but I'm not sure how to nicely expand it in the same way that it is in Array. Maybe this'll do for now.
```suggestion
				Creates a typed dictionary from the [param base] dictionary. A typed dictionary can only contain keys and values of the given types, or that inherit from the given classes, as described by this constructor's parameters.
```